frontend/src/pages/pipelines/global/modelCustomization/FineTunePageFooter.tsx

```diff
@@ -0,0 +1,49 @@
+import { ActionList, ActionListItem, Button, Stack, StackItem } from '@patternfly/react-core';
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+type FineTunePageFooterProps = {
+  isStartRunDisabled: boolean;
+  link: string;
```

Comment:
lets make this have a more clear name. maybe we expose a onSuccess func in the params and we do the navigate from the parent component